vividus-to-xray-exporter/src/main/java/org/vividus/xray/exporter/XrayExporter.java

```diff
@@ -64,7 +66,7 @@
     @Autowired private TestCaseFactory testCaseFactory;
     @Autowired private TestExecutionFactory testExecutionFactory;
 
-    private final List<ErrorExportEntry> errors = new ArrayList<>();
+    private final List<Supplier<String>> errors = new ArrayList<>();
```

Comment:
overhead

Reply:
why not? no reason to generate bunches of strings eagerly